python/lsst/ts/m2com/mock_command.py

```diff
@@ -24,14 +24,26 @@
 from lsst.ts import salobj
 from lsst.ts.idl.enums import MTM2
 
-from . import DetailedState, CommandStatus
+from . import DetailedState, CommandStatus, CommandScript, PowerType, DigitalOutput
 
 
 __all__ = ["MockCommand"]
 
 
 class MockCommand:
-    """Mock command to simulate the execution of command in real hardware."""
+    """Mock command to simulate the execution of command in real hardware.
+
+    Parameters
+    ----------
+    is_csc : `bool`, optional
+        Is called by the commandable SAL component (CSC) or not. (the default
+        is True)
+    """
+
+    def __init__(self, is_csc=True):
```

Comment:
This is intensional to avoid multiple classes to update/write the `model` object in the same time. Right now, only the `MockServer` will hold the `MockModel` as an internal object. Because the `asyncio` is used in `MockServer`, if I let the `MockCommand` to hold the same object of `MockModel` in `MockServer`, I may need to add the `asyncio.Lock()` as well. I want to avoid this to make the resource management easier. Thanks!